content/chapters/app-interact/lab/support/time-server/server.c

```diff
@@ -7,49 +8,36 @@
 #include <time.h>
 #include <unistd.h>
 
+#include "../utils/utils.h"
+
 #define BIND_ADDR "0.0.0.0"
 #define PORT 2000
 
-int create_socket(char *addr, short port)
+int create_socket(char *addr)
```

Comment:
I think the port should be left as function argument.

Reply:
Commited an unintended change.